bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java

```diff
@@ -58,64 +68,65 @@ public static String normalizePhoneNum(String phoneNum) throws TskCoreException
 	 * @param emailAddress The email address string to be normalized.
 	 *
 	 * @return The normalized email address.
-	 * @throws TskCoreException If the given string is not a valid email address.
+	 *
+	 * @throws TskCoreException If the given string is not a valid email
+	 *                          address.
 	 */
 	public static String normalizeEmailAddress(String emailAddress) throws TskCoreException {
-		
-        if (isValidEmailAddress(emailAddress)) {
-            return emailAddress.toLowerCase();
-        } else {
-            throw new TskCoreException(String.format("Input string is not a valid email address: %s", emailAddress));
-        }
+		if(!StringUtils.isBlank(emailAddress)) {
+			return emailAddress.toLowerCase();
+		} else {
+			return emailAddress;
+		}
 	}
-	
+
 	/**
-	 * Checks if the given accountId is a valid id for 
-	 * the specified account type.
-	 * 
-	 * @param accountType Account type.
+	 * Checks if the given accountId is a valid id for the specified account
+	 * type.
+	 *
+	 * @param accountType     Account type.
 	 * @param accountUniqueID Id to check.
-	 * 
-	 * @return True, if the id is a valid id for the given account type, False otherwise.
+	 *
+	 * @return True, if the id is a valid id for the given account type, False
+	 *         otherwise.
 	 */
 	public static boolean isValidAccountId(Account.Type accountType, String accountUniqueID) {
 		if (accountType == Account.Type.PHONE) {
 			return isValidPhoneNumber(accountUniqueID);
-		}
-		if (accountType == Account.Type.EMAIL) {
+		} else if (accountType == Account.Type.EMAIL) {
 			return isValidEmailAddress(accountUniqueID);
+		} else {
+			return !StringUtils.isEmpty(accountUniqueID);
 		}
-		
-		return !StringUtils.isEmpty(accountUniqueID);
 	}
-	
+
 	/**
-	 * Checks if the given string is a valid phone number.
+	 * Checks if the given string is a valid phone number. A valid phone number
+	 * is one that has a non-zero number of digits in it.
 	 *
 	 * @param phoneNum Phone number string to check.
 	 *
-	 * @return True if the given string is a valid phone number, false otherwise.
+	 * @return True if the given string is a valid phone number, false
+	 *         otherwise.
 	 */
 	public static boolean isValidPhoneNumber(String phoneNum) {
-		if (!StringUtils.isEmpty(phoneNum)) {
-			return phoneNum.matches("\\+?[0-9()\\-\\s]+");
+		if (!StringUtils.isBlank(phoneNum)) {
+			// Any number of digits is good enough.
+			return !phoneNum.replaceAll(NON_DIGITS, "").isEmpty();
```

Comment:
I'm afraid this definition of a valid phone number is wrong.  

According to this, each of these is a valid phone number:
"1+1"
"! 1      3"
"2 #!!!!!%$#@ 3"
"2 #!sdaswfdsafasdfasda!!!!%$#@ "

I can understand our desire to relax rules for length (the old method didn't apply any length restrictions) , and allow some telephony signs (+ * # are the only valid chars allowed in a sequence of phone digits), but this applies no rules at all. 

We need to bring this back to functional parity with what was (leading +, digits, parenthesis, spaces), and allow '*' & '#'.

Reply:
Well these were all valid with the previous regex:
+-----
()()()()
-         -
)))))

At least now we know there are digits. Having worked with validating phone numbers before I can tell you its a rabbit hole.

Reply:
Fair enough.  The input to this isn't totally junk , so its not likely any of above examples would be found that need to be validated as phone numbers.

The new method, however also says "dsmyda01@gmail.com" is a valid phone number.

The kind of input strings the CVT and communication artifact viewers  need to handle look like these:

5978620523@s.whatsapp.net
8:live:.cid.8f278e2e9be3ae57
128
bba1b3f8a10533b9	
kysm-g920v06157df651d9371e509fff3e650ff8d0	

Each of them is being reported as a valid phone number when they aren't.  

This change breaks the Artifact content viewers.